added more boilerplate on pull request template

fernandovgs opened the pull request:
### **Description**
Adds more boilerplate on file pull_request_template.md, in order to make a better description for pull requests

### **Closes issue(s)**
#146 

### **Test / reproduce**

### **Screenshots**

### **Changes include**
- [ ] Bugfix (non-breaking change that solves an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (change that is not backwards-compatible and/or changes current functionality)
- [x] Documentation update

### **Checklist**
- [ ] I have tested this code
- [ ] I have updated the README
- [ ] My code follows the code style of this project

### Other comments

Commit:
added more infos on pull request template

joaofavoretti commented:
I just mentioned about that on the Issue #146 about that. What do you think about changing the file name to PULLREQUEST_TEMPLATE.md just for standardization.

joaofavoretti commented:
I was also thinking about leaving some explanation for each title, and an example as well.

joaofavoretti commented on docs/pull_request_template.md:
```diff
@@ -1,7 +1,20 @@
-Fixes # .
+### **Description**
```
I think you should add here something like:

```
Here it will be a general overview of the changes involved in your pull-request. It allows contributors to piece each modification together while reviewing it.

Example of usage:

This PR proposes the following changes:
-
-
```

fernandovgs replied:
Last commit added this text!

joaofavoretti commented:
I have some other ideas for the rest of the fields. But let me know if you agree with that kind of information for each of the tópics listed.

fernandovgs commented:
> I have some other ideas for the rest of the fields. But let me know if you agree with that kind of information for each of the tópics listed.

I totally agree. The more info, the better.
What about the other ideas you mentioned?

Commit:
doc: standardized PR template name

Commit:
doc: improvements on PR template

fernandovgs commented:
Added descriptions on each section of PULL_REQUEST_TEMPLATE.md. Also, updated CONTRIBUTING.md as requested [here.](https://github.com/courselab/snaskii21/issues/146#issuecomment-1011281360).

joaofavoretti commented:
That is great!!

Yudiaramos reviewed (approved):
other than the requested changes it looks great to me!

Commit:
doc: added more info on pr template
